jvb/src/main/kotlin/org/jitsi/videobridge/octo/OctoRelayServiceProvider.kt

```diff
@@ -21,7 +21,11 @@ import org.jitsi.videobridge.octo.config.OctoConfig
 class OctoRelayServiceProvider {
     private val octoRelayService: OctoRelayService? by lazy {
         if (OctoConfig.config.enabled) {
-            OctoRelayService()
+            try {
+                OctoRelayService()
+            } catch (t: Throwable) {
+                null
```

Comment:
Should we log an error?

Reply:
We don't have a logger here, so I logged it in `OctoRelayService`